webservices/common/models/itemized.py

```diff
@@ -652,7 +652,8 @@ class ScheduleE(PdfMixin, BaseItemized):
 
 
 class ScheduleEEfile(BaseRawItemized):
-    __tablename__ = 'real_efile_se_f57_vw'
+    # __tablename__ = 'real_efile_se_f57_vw'
+    __tablename__ = 'real_efile_se_f57_vw_tmp_pk'
 
```

Comment:
I think we should use real view, not the _tmp

Reply:
Thanks @hcaofec for reviewing. This is a WIP PR and not ready for review yet. Paul is testing the  SE efile new tsvector filter on his local off my feature branch. I will update and upload the latest view after i get a sign off from Paul.